File: problems/algorithms/function-prime-nth/ruby/response/prime-nth.rb
require_relative '../../../function-prime/ruby/response/src/function_prime.rb'

# begin: 1..n
# end: 1..n, end > begin

def nthPrime(nth)
  result = 2
  count = 1
  number = 3
  while count < nth
    if isPrime(number)
      result = number
      count += 1
    end
    number += 1
  end
  return result
end

File: src/pages/exercises/function-prime-nth/_codes/js/response/prime-nth.js
import { isPrime } from '../../../../function-prime/_codes/js/response/prime.js';

/**
 * begin: 1..n
 * end: 1..n, end > begin
 */
export function nthPrime(nth) {
  let result = 2;

  for (let number = 3, count = 1; count < nth; number++) {
    if (isPrime(number)) {
      result = number;
      count++;
    }
  }

  return result;
}
